cypress/integration/models/administrator/credentials/credentialsSourceControlUsername.ts

```diff
@@ -29,11 +30,11 @@ export class CredentialsSourceControlUsername extends CredentialsSourceControl {
     }
 
     protected fillUsername() {
-        inputText("#user", this.username);
+        inputText(usernameInput, this.username);
```

Comment:
Why is this required in this PR ?

Reply:
The test case fails because of this locator